addons/kommander/1.2.0/kommander-12.yaml

```diff
@@ -0,0 +1,98 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: kommander
+  labels:
+    kubeaddons.mesosphere.io/name: kommander
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.2.0-11"
```

Comment:
I believe this has to be bumped as well.

Reply:
this one needs to be bumped, yes.

Reply:
```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: "1.2.0-12"
```